pkg/server/apis/v1alpha1/descriptors/project.go

```diff
@@ -151,4 +151,27 @@ var project = []definition.Descriptor{
 			},
 		},
 	},
+	{
+		Path:        "/projects/{project}/cleancachetasks",
```

Comment:
```suggestion
		Path:        "/projects/{project}/cleancache",
```

Reply:
I think `cleancachetasks`, a noun, is more RESTful than `cleancache`, a verb.